oshi-core/src/main/java/oshi/hardware/platform/mac/MacGlobalMemory.java

```diff
@@ -108,7 +108,7 @@ public List<PhysicalMemory> getPhysicalMemory() {
                 }
             }
         }
-        pmList.add(new PhysicalMemory(bankLabel, capacity, speed, manufacturer, memoryType));
+        pmList.add(new PhysicalMemory(bankLabel, capacity, speed, manufacturer, memoryType, Constants.UNKNOWN));
```

Comment:
Looks like we can look for the part number here, see https://serverfault.com/questions/1081248/how-can-i-get-ram-frequency-from-the-os-x-low-level-terminal

Reply:
I have only w10 and linux Mint, currently im working on w10 wit my project so I have extend memorychip by partnumber only for windows systems. I will not extend other systems because of lack of knowledge - even i will fond something online i will have no opportunity to test it on my lap. That is the reason of Constants.UNKNOWN.